src/molecules/CardReadOnly/ReadOnlyCard.jsx

```diff
@@ -3,30 +3,38 @@ import { Card } from "../Card/Card";
 import { CardContent } from "../CardContent/CardContent";
 import { CardImage } from "../CardImage/CardImage";
 import { dateChanged, diffDateDisplay } from "../../util/date";
+import { FolderStar } from "../FolderStar";
 
 export const ReadOnlyCard = ({
+  id,
   url,
-  imageSource,
+  image_source,
   alt,
   elapsedTime,
   description,
-  createdAt,
+  created_at,
 }) => {
   const [isHovered, setIsHovered] = useState(false);
   const handleMouseOver = () => setIsHovered(true);
   const handleMouseLeave = () => setIsHovered(false);
```

Comment:
```suggestion
  const hoverCard = () => setIsHovered(prev => !prev);
```

이 두 함수는 위와 같이 하나의 함수로 구성할 수 있겠습니다.
state의 이전 값을 가져와서 반대연산자를 이용해 해당 값의 반대 값을 다시 할당하는 방식으로 구현됩니다.